test/validation/ferc1_test.py

```diff
@@ -23,11 +40,42 @@ def test_pu_ferc1(pudl_out_ferc1):
 
 @pytest.mark.ferc1
 @pytest.mark.post_etl
-def test_steam_ferc1(pudl_out_ferc1):
+def test_steam_ferc1_trivial(pudl_out_ferc1):
     """Test output routines for tables from FERC Form 1."""
     logger.info("Compiling FERC Form 1 steam plants table...")
-    logger.info(f"{len(pudl_out_ferc1.plants_steam_ferc1())} "
-                f"steam plant records found.")
+    logger.info(
+        f"{len(pudl_out_ferc1.plants_steam_ferc1())} steam plant "
+        f"records found."
+    )
+
+
+@pytest.mark.ferc1
+@pytest.mark.post_etl
+@pytest.mark.xfail
```

Comment:
But then also @cmgosnell I added this test to flag any cases where the PUDL Plant IDs and FERC Plant IDs are in conflict with each other... which they currently are (which is why it is marked `xfail` meaning it is expected to fail) which will be resolved by #241 which is assigned to you.